docs/README.md

```diff
@@ -1,328 +0,0 @@
-### Table of Contents

```

Comment:
upstream symblinked this to the btcd/README.md. we should do the same, but remember to replace main README with this one.